src/main/scala/microsites/MicrositeKeys.scala

```diff
@@ -167,14 +201,64 @@ trait MicrositeAutoImportSettings extends MicrositeKeys {
 
   lazy val fr = new FileReader
 
+  def createVersionsJson(targetDir: String, content: List[Version]): File = {
+    val jekyllDir  = "jekyll"
+    val targetPath = s"$targetDir$jekyllDir/_data/versions.json"
+    createFile(targetPath)
+    writeContentToFile(content.asJson.toString, targetPath)
+    targetPath.toFile
+  }
+
+  def generateVersionList(versionStringList: List[String], ownVersion: String) = {
+    versionStringList
+      .map(version => {
+        Version(version, own = (ownVersion == version))
+      })
+      .toList
+  }
+
+  def pluginProjects(pluginName: String): Option[Array[String]] = {
+    val sbtPluginsOutput = "sbt --error plugins".lineStream
+    val pluginLine =
+      sbtPluginsOutput.find(line => line.trim.startsWith(s"$pluginName: enabled in "))
```

Comment:
```suggestion
      sbtPluginsOutput.find(_.trim.startsWith(s"$pluginName: enabled in "))
```

Reply:
Ha!  And I was asking myself how could I use that underscore syntax... 😅